gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -15,8 +15,10 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
     }
 
     private AndroidCommandPluginExtension createExtension() {
-        def project = ProjectBuilder.builder().build()
+	def projectDir = new File('..')
+        def project = ProjectBuilder.builder().withProjectDir(projectDir).build()
```

Comment:
:x: While I understand this might fix the tests in our situation I agree with @friedger that this is not clean solution. 
Can't we use a nice closure to find the directory that contains the local.properties?
Also: If we move the base for the project dir one level up, are there any problems with other code/relative settings?